content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/source/touchui-composite-multifield.js

```diff
@@ -0,0 +1,303 @@
+                                                    sling:resourceType="granite/ui/components/foundation/layouts/fixedcolumns"
+                                                    method="absolute"/>
+                                                    <items jcr:primaryType="nt:unstructured">
+                                                        <column
+                                                        jcr:primaryType="nt:unstructured"
+                                                        sling:resourceType="granite/ui/components/foundation/container">
+                                                            <items jcr:primaryType="nt:unstructured">
+                                                                <page
+                                                                jcr:primaryType="nt:unstructured"
+                                                                sling:resourceType="granite/ui/components/foundation/form/textfield"
+                                                                fieldDescription="Name of Page"
+                                                                fieldLabel="Page Name"
+                                                                name="./page"/>
+                                                                <path
+                                                                jcr:primaryType="nt:unstructured"
+                                                                sling:resourceType="granite/ui/components/foundation/form/pathbrowser"
+                                                                fieldDescription="Select Path"
+                                                                fieldLabel="Path"
+                                                                name="./path"
+                                                                rootPath="/content"/>
+                                                            </items>
+                                                        </column>
+                                                    </items>
+                                                </field>
+                                            </pages>
+                                        </items>
+                                    </column>
+                                </items>
+                            </fieldset>
+                        </items>
+                    </column>
+                </items>
+            </content>
+        </jcr:root>
+     </code>
+*/
+
+
+(function () {
+    var DATA_ACS_AEM_NESTED = "data-acs-aem-nested",
+        CFFW = ".coral-Form-fieldwrapper",
+        _ = window._, CUI = window.CUI, Class = window.Class;
+
+    //reads multifield data from server, creates the nested composite multifields and fills them
+    function addDataInFields() {
+        $(document).on("dialog-ready", function() {
+            var mName = $("[" + DATA_ACS_AEM_NESTED + "]").data("name"),
```

Comment:
hmm, mName is a plain string, here its multifield name say "./pages" , a variable prefixed with $ is output of some jquery selector, you know to easily identify in the code, can apply jquery functions on the variable without doing $() again... i didn't get the "."